.github/maintainers_guide.md

```diff
@@ -40,53 +40,54 @@ Reference docs are built by using various open source tools and formats. This in
 
-Releasing can feel intimidaitng at first, but rest assured: if you make a mistake, don't fret! npm allows you to unpublish a release within the first 72 hours of publishing (you just won’t be able to use the same version number again). Venture on!
+Releasing can feel intimidating at first, but rest assured: if you make a mistake, don't fret! npm allows you to unpublish a release within the first 72 hours of publishing (you just won’t be able to use the same version number again). Venture on!
 
 1. Check the status of the package's GitHub Milestone for issues that should be shipped with the release.
 
     - If all issues have been closed, continue with the release. 
     
     - If issues are still open, discuss with the team about whether the open issues should be moved to a future release or if the release should be held off until the issues are resolved.
 
-2. Navigate to the specific package you're releasing in the `packages/` directory.
+2. Make sure your local `main` branch has the latest changes. 
 
-3. Run `npm run test && npm run lint` to verify that everything is working and free of linting errors.
+3. Navigate to the specific package(s) you're releasing in the `packages/` directory.
 
-4. Bump the version in `package.json` (see [Versioning and Tags](https://github.com/slackapi/node-slack-sdk/blob/main/.github/maintainers_guide.md#versioning-and-tags))
+4. For each package to be released, run `npm run test && npm run lint` to verify that tests are passing and code is free of linting errors.
 
-    - Make a single commit for the version bump ([Example](https://github.com/slackapi/node-slack-sdk/commit/1503609d79abf035e9e21bad7360e124e4211594))
+5. Bump the version(s) in `package.json` (see [Versioning and Tags](https://github.com/slackapi/node-slack-sdk/blob/main/.github/maintainers_guide.md#versioning-and-tags))
+
+    - Make a single commit for the version(s) bump, following the format in: ([Example](https://github.com/slackapi/node-slack-sdk/commit/1503609d79abf035e9e21bad7360e124e4211594))
 
     - Create a pull request for the version change ([Example](https://github.com/slackapi/node-slack-sdk/pull/1059))
 
-    - Add appropriate labels, including `release`
+    - Add appropriate labels on the pr, including `release`
 
-5. Once the PR has been approved and tests have passed, merge it into the main repository.
+6. Once the PR has been approved and tests have passed, merge it into the main repository.
 
     -  Update your local main branch: `git pull origin main`
 
     -  Add a version tag (ie, `git tag @slack/web-api@5.6.0`)
 
     -  Push the new tag up to origin: `git push --tags origin`
 
-6. Publish the release to npm
+7. Publish the release to npm
+    - To publish, you need to be a member of the `slack Org` on npm and set up 2-Factor Auth with your passsword generator of choice. Before you can publish with npm, you must run `npm login` from the command line.
```

Comment:
I'm torn about this being here, as this is more of a miss in our onboarding process than steps on how to publish, but I'm OK with including it here if you find it useful.